app/posts/claim-general-mentor-training-funding/2022-06-26-general-mentor-training-grant-guidance-and-claim-form.md

```diff
@@ -0,0 +1,123 @@
+
+![Google form confirmation the form has been completed](google-form/05-confirmation.png "Claim form — confirmation")
+
+### Information required for a claim
+
+#### School details
+
+So we know who is making the claim and who to pay.
+
+The amount of funding available is adjusted for location. To pay the correct amount DfE will need to know where the school is.
+
+| **Location**    | **Amount** |
+|:---------------:|:----------:|
+| Inner London    |      £1072 |
+| Outer London    |       £965 |
+| Fringe          |       £902 |
+| Rest of England |       £876 |
+
+This also allows us to check compliance using data from [Register trainee teachers](https://www.register-trainee-teachers.service.gov.uk/) (Register). Schools can only claim for mentors who mentor at least one trainee each. Assuming Register collects placement data by the end of the 2024/25 academic year then we can verify that the school has not claimed for more teachers than it is eligible to.
+
+#### Number of general mentors
+
+This is a basic check on the claim. Schools can claim a maximum of 20&nbsp;hours per trainee. It would also help the DfE get an understanding of how many general mentors there are.
+
+#### Hours of classroom cover so the mentor could train
+
+The aim of the grant is to help schools pay for cover so the mentor isn’t compelled to train out of school hours.
+
+To simplify the grant, schools are claiming for hours of classroom cover. Rather than specifying the costs that may have been incurred.
+
+#### Hours of classroom cover paid for by 
+
+To prevent paying schools for something already paid for, the DfE wants to deduct any hours of classroom cover paid for by the ITT provider.
+
+The form could ask for ‘hours of backfill not paid for by the provider’. Our assumption is that it's easier to ask for the cover and the cover paid for by the provider separately instead of asking the user to make the calculation.
+
+#### Declaration that each mentor mentored at least one trainee each
+
+Funding is only available to mentors who mentored a trainee in the 2024/25 academic year. The minimum ratio of trainees to mentors is 1:1. For example, if a trainee had 2 mentors and neither of those mentors mentored another trainee, then the school could only claim for one of those mentors.
+
+#### Email address of user
+
+So the DfE can contact the user about their claim.
+
+### Assumptions and questions to test
+
+The main things we wanted to understand were:
+
+- participants’ reaction to new training requirement, the impact that would have and how the funding was perceived
+
+- school’s use of cover and anticipated costs
+
+- participants’ understanding and reaction to the rules of the funding
+
+- what funding placement schools get from providers and if this is specifically for time when the trainee
+
+- how participants would expect to be made aware of their entitlement to claim this funding
+
+- their ability to get hold of the data needed to make a claim
```

Comment:
Is it intentional that these bullets are more spaced than the bullets further up the page?